File: problemas/ad-hoc/2137/2137.cpp
#include <algorithm>
#include <iostream>

using namespace std;

int main(){
    int N;
    string livros[1001];

    while(cin >> N){
        for(int i = 0; i < N; ++i){
            cin >> livros[i];
        }

        sort(livros, livros + N);

        for(int i = 0; i < N; ++i){
            cout << livros[i] << endl;
        }
    }

    return 0;
}

File: problemas/strings/2137/2137.ts
process.stdin.resume();
process.stdin.setEncoding('utf8');

let input = '';
let lineCount = 0;

process.stdin.on('data', (chunk) => {
  input += chunk;
});

process.stdin.on('end', () => {
  var lines = input.split('\n');

  while (lines.length) {
    let N = parseInt(lines.shift()!);

    let livros = [];
    for (let i = 0; i < N; ++i) {
      livros.push(lines.shift()!.trim());
    }

    livros.sort();

    for (let i = 0; i < N; ++i) {
      console.log(livros[i]);
    }
  }
});